frontend/packages/ceph-storage-plugin/src/components/kms-config/utils.tsx

```diff
@@ -66,28 +65,27 @@ export const generateClientKeySecret = (clientKey: string) => ({
   },
 });
 
-export const createKmsResources = (kms: KMSConfig, update = false, previousData?: any) => {
-  let tokenSecret: SecretKind;
-  if (kms.token) {
-    tokenSecret = {
-      apiVersion: SecretModel.apiVersion,
-      kind: SecretModel.kind,
-      metadata: {
-        name: KMSSecretName,
-        namespace: CEPH_STORAGE_NAMESPACE,
-      },
-      stringData: {
-        token: kms.token.value,
-      },
-    };
-  }
-
-  const resources: Promise<K8sResourceKind>[] = [];
+export const createAdvancedKmsResources = (kms: KMSConfig) => {
+  const advancedKmsResources: Promise<K8sResourceKind>[] = [];
+
+  if (kms.caCert)
+    advancedKmsResources.push(k8sCreate(SecretModel, kms.caCert, { ns: CEPH_STORAGE_NAMESPACE }));
```

Comment:
```suggestion
    advancedKmsResources.push(k8sCreate(SecretModel, kms.caCert));
```

I think this can be done for all such instances. Refer: https://github.com/openshift/console/pull/9623